cy43439.go

```diff
@@ -613,6 +662,10 @@ func (d *Device) wifiOn(country uint32) error {
 	return nil
 }
 
+func (d *Device) SetMAC(mac [6]byte) error {
+	return d.WriteIOVarN("cur_etheraddr", whd.WWD_STA_INTERFACE, mac[:])
```

Comment:
should also set the mac field if it succeeds, right?

Reply:
Yes, oops.  I think I'll remove this SetMAC func for now, since we don't need it.  Later, if we add a SetHardwareAddr() method to Netlinker, we can revisit.